GVFS/GVFS.Upgrader/GVFS.Upgrader.csproj

```diff
@@ -34,7 +39,19 @@
         </Compile>
       </ItemGroup>
     </When>
-    <Otherwise>
+    <When Condition="'$(IsLinux)' == 'true'">
+      <PropertyGroup>
+        <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+        <RuntimeIdentifiers>linux-x64</RuntimeIdentifiers>
+      </PropertyGroup>
+      <ItemGroup>
+        <ProjectReference Include="..\GVFS.Platform.Linux\GVFS.Platform.Linux.csproj" />
+        <Compile Include="..\GVFS.PlatformLoader\PlatformLoader.Linux.cs">
+          <Link>PlatformLoader.Linux.cs</Link>
+        </Compile>
+      </ItemGroup>
+    </When>
+    <When Condition="'$(IsOSX)' == 'true'">

```

Comment:
This `When` is also broken in VS for Mac and needs to be switched back to `Otherwise`

Reply:
:+1: